01/app.js

```diff
@@ -0,0 +1,27 @@
+const formEl = document.querySelector('form')
+
+formEl.addEventListener('submit', function (e) {
+	e.preventDefault()
+	getUser(e)
+})
+
+function getUser(e) {
+	const firstName = e.target.elements.firstName
+	const lastName = e.target.elements.lastName
+
+	if (firstName.value !== '' && lastName.value !== '') {
+		addUser(firstName, lastName)
+	} else {
+		alert('Wprowadź wszystkie dane')
+	}
+}
+
+function addUser(firstName, lastName) {
+	const usersList = document.querySelector('.users-list')
+	const newUser = document.createElement('li')
+	newUser.classList.add('users-list__person')
+	newUser.textContent = `${firstName.value} ${lastName.value}`
+	usersList.appendChild(newUser)
+	firstName.value = ''
+	lastName.value = ''
```

Comment:
Super, że jest funkcja! - zwróć uwagę jednak na to co one robi i jak się nazywa.
Funkcja mówi o dodawaniu użytkownika, a jeszcze czyści pole formularza (to 2 czynności).
W ten sposób łatwo zapomnieć, że ta funkcja coś więcej robi (a nie powinna).
Lepiej rozdzielić to na 2 osobne funkcje - będzie czytelniej i nie będzie "ukrytych" działań, które w dłuższym odstępie czasu będą zapominanie.